### PR TITLE
Updates to multiple subpages for imaging

### PR DESCRIPTION
**Pull request by horsto**

### CODE REVIEW

File: docs/common/getting_started/python.md [hunk 1]
```diff
@@ -52,43 +52,16 @@ The Ephys and Imaging pipelines have subtly different requirements. Therefore, w
...
-In the latter case, you must change `spyder` Preferences to use the appropriate Python interpreter. You can find the correct path by running the following code _inside the environment you wish to use_:
-
-```python
-python -c "import sys; print(sys.executable)"
-```
-
-And then copying this path to the provided textbox in `Preferences > Python Interpreter > Use the following interpreter`
-
-Numerous other IDEs support Python. Common examples include PyCharm and Visual Studio Code
+  * `conda activate ephys` (or `conda activate imaging`)
```
Comment (simon-ball) [inline docs/common/getting_started/python.md]: Not necessary, Jupyter can (and, imo, should) be installed in the one environment only, preferably `base`, with other environments made available via `ipykernel` and `nb_conda_kernels`. It's pretty much the _only_ package that I strongly recommend be installed in the `base` environment only

Installing and running Jupyter from inside a single environment is both inefficient (i.e. multiple copies installed), inflexible (you need an independent Jupyter server for each environment you're using), and worst of all, actively encourages bad practice (the annoyance of switching from one server to another encourages users to stick to a single environment).

File: docs/imaging/Folder-logic.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ The **2Pminiscope_A** import logic is as follows:
 - The base folder contains raw scanimage .tif files that follow a certain naming scheme:
     - `basename_filenumber_filenumber.tif`
 
-- Wavesurfer synchronisation files (`basename_session.h5`)
+- Wavesurfer synchronisation files (`basename_session.h5`)x
```
Comment (simon-ball) [inline docs/imaging/Folder-logic.md]: typoed extra `x`?

File: docs/common/getting_started/python.md (hunk 1 above)
Comment (horsto) [reply docs/common/getting_started/python.md]: Updated in commit 6b7a4c2

File: docs/imaging/Folder-logic.md (hunk 2 above)
Comment (horsto) [reply docs/imaging/Folder-logic.md]: Fixed in 1c91b57

**Comment by horsto** [comment]
Ok to merge? @simon-ball

**Pull request merged**